service.yml

```diff
@@ -13,10 +13,12 @@ semaphore:
   maven_skip_deploy: true
   build_arm: true
   nano_version: true
+  branches: []
+  triggers: []
 
 # Properties not ported from Jenkinsfile:
 #
 # upstreamProjects = 'confluentinc/rest-utils' --- used "downstream_repos in rest-utils
 # cpImages = true
 # osTypes = ['ubi8']
-# slackChannel = 'ksqldb-warn'
\ No newline at end of file
+# slackChannel = 'ksqldb-warn'
```

Comment:
@lucasbru -- while semaphor is not running yet, this slack channel seems to be outdate? -- We we re-enable is, can we also update the slack channel?